Sell/Product/Repository/ProductRepository.swift

```diff
@@ -8,18 +8,18 @@
 
 import Foundation
 
-final class ProductRepository {
+public final class ProductRepository {
 
     private let dispatcher: GraphQLDispatching
 
-    init(dispatcher: GraphQLDispatching) {
+    public init(dispatcher: GraphQLDispatching) {
```

Comment:
I don't think we need to make this public, unless someone outside the SDK is going to be allowed to make instances.